README.md

```diff
@@ -24,34 +24,39 @@ To run the bootstrap task, use the bolt command, specifying the PE master from w
 
 #### Example: Basic usage
 
-On the command line, run `bolt task run bootstrap master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+On the command line:
+* For Linux,   run `bolt task run bootstrap::linux   master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+* For Windows, run `bolt task run bootstrap::windows master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+For all advanced examples below, simply replace `bootstrap::linux` by `bootstrap::windows` to perform the action on Windows.
 
 #### Example: Verify the master's CA on initial connection
 
-Optionally to validate the connection during the boostrap process, specify the puppet master's CA cert: `bolt task run bootstrap master=<master's fqdn> cacert_content="$(cat /etc/puppetlabs/puppet/ssl/certs/ca.pem)" --nodes x,y,z --modules /path/to/modules`
+Optionally to validate the connection during the bootstrap process, specify the puppet master's CA cert by adding the cacert_content option:
+`bolt task run bootstrap::linux master=<master's fqdn> cacert_content="$(cat /etc/puppetlabs/puppet/ssl/certs/ca.pem)" --nodes x,y,z --modules /path/to/modules`
```

Comment:
If you really want these on a separate line, need to add an extra newline. They'd probably format better with triple ticks, as in
```
bolt task ...
```

Reply:
Missing example for using on Windows i.e. `cacert_content="$(cat /etc/puppetlabs/puppet/ssl/certs/ca.pem)"` makes no sense on a Windows platform